lib/check-minor-version.js

```diff
@@ -0,0 +1,18 @@
+var semver = require('semver')
+
+module.exports = function (app) {
+  if (!app || !app.path) return
+
+  require('electron-version')(app.path || 'electron', function (err, electronVersion) {
```

Comment:
> Do you think it's worth it?

Hmm, since this file is called by `Application.start` which is used by every consumer of Spectron, it would seem like it would always be required so eagerly requiring it should be fine. Same with the `require('./check-minor-version')(this)` line in `application.js`. I think it would be fine/clearer to assign that one at the top as well.